src/pages/signIn/signUp.jsx

```diff
@@ -0,0 +1,90 @@
+import { Formik } from 'formik';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { getUserInfo } from '../../redux/actions/actionCreators';
+import { initValuesForSignUp, ValidSchemeForSignUp } from './validation';
+import { SignUpForm, SignForm, SignInButton, StyledLink, StyledTitle } from './styled';
+import ValidInput from './validInput';
+import { useMutationDataServer } from '../../hooks/mutation/dataServerMutation';
+
+const SignUp = () => {
+    const dispatch = useDispatch();
+
+    const mutation = useMutationDataServer();
+
+    const loadDataToServer = (dataToSend) => {
+        dispatch(getUserInfo(dataToSend));
+        mutation.mutate(dataToSend);
+    };
+
+    return (
+        <Formik
+            initialValues={initValuesForSignUp}
+            validationSchema={ValidSchemeForSignUp}
+            onSubmit={async (values) => {
+                const dataToSend = {
```

Comment:
you don;t need to create the object since you post all data, you can just do like this loadDataToServer(values);